types/react-toastr/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for react-toastr 3.0
+// Type definitions for react-toastr 3.1
```

Comment:
I think the intent is for this version number to be `react-toastr`'s version number, which is still 3.0.0. Someone who maintains DefinitelyTyped can confirm.